### ISSUE TITLE
tidy up imported packages orders

### ISSUE THREAD

**Issue by yaruwangway** [cosmos/gaia#1527]
## Summary
tidy the imported packages order:

```
import (
stdlib

cosmos-sdk/tendermint/ibc others

gaia lib
)
```

## Proposal

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned
- [ ] Is a spike necessary to map out how the issue should be approached?

**Comment by Anmol1696** [comment]
Can to take this up, if this is still needed for rho upgrade.

**Comment by yaruwangway** [comment]
feel free ! Thank you ! @Anmol1696 👍

**Comment by mmulji-ic** [comment]
@Anmol1696 do you want to run with this?

**Comment by okwme** [comment]
hey @Anmol1696 is this included in #1580

**Comment by Anmol1696** [comment]
Ahh sorry missed out on the thread, can add this to #1580 as well if you guys want, but i will raise another PR today for this
